app/di.scala

```diff
@@ -2,19 +2,27 @@ import controllers._
 import play.api.{Application, ApplicationLoader, BuiltInComponentsFromContext, Logger}
 import play.api.ApplicationLoader.Context
 import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.mvc.EssentialFilter
 import play.api.routing.Router
+import play.filters.HttpFiltersComponents
 import services.{AWS, LogShipping}
 import router.Routes
 
-
-
 class AppLoader extends ApplicationLoader {
   def load(context: Context): Application = {
     new AppComponents(context).application
   }
 }
 
-class AppComponents(context: Context) extends BuiltInComponentsFromContext(context) with AhcWSComponents {
+class AppComponents(context: Context)
+  extends BuiltInComponentsFromContext(context)
+    with AhcWSComponents
+    with AssetsComponents
+    with HttpFiltersComponents {
+
+  // disabling some default filters until value is clear in this context
+  override def httpFilters: Seq[EssentialFilter] =
+    super.httpFilters.filterNot(filter => filter == csrfFilter || filter == allowedHostsFilter)
```

Comment:
My thinking is that the app will be no less secure than it is now with these filters disabled, it's clear that they are disabled, and I wanted to make sure that I haven't broken anything before enabling these filters one by one.  The CSRF filter looks like there will be quite a few changes involved.